hudi-common/src/main/java/org/apache/hudi/metadata/HoodieTableMetadataUtil.java

```diff
@@ -1364,24 +1342,47 @@ public static HoodieData<HoodieRecord> convertMetadataToColumnStatsRecords(Hoodi
     }
   }
 
+  public static List<String> getColumnsToIndex(HoodieCommitMetadata commitMetadata,
```

Comment:
Though this is getting exercised in the functional tests in `TestColumnStatsIndex`, it would be nice to make it visible for testing and add a UT for this method in `TestHoodieTableMetadataUtil` if possible.